RecCalorimeter/src/components/CorrectCaloClusters.h

```diff
@@ -81,6 +85,16 @@ class CorrectCaloClusters : public GaudiAlgorithm {
                                      std::vector<std::vector<double>> parameters,
                                      const std::string& funcNameStem = "upDown");
 
+  /**
+   * Initialize vectors of benchmark correction functions.
+   *
+   * @return                  Status code.
+   */
```

Comment:
Can this function have the same name? it operates on vector of functions instead of vector of vector.